src/transac/components/transactions/transactions.service.coffee

```diff
@@ -0,0 +1,255 @@
+###
+#   @desc Backend Service for Transactions & Matches.
+###
+angular.module('transac.transactions').service('TransacTxsService', ($log, $http, $q, $window, TransacUserService, DEV_AUTH)->
+
+  _self = @
+
+  _self.HTTP_CONFIG = {}
```

Comment:
By convention, the constants are in uppercase. This is not a constant, it should be camel cased.